app/Http/Controllers/FrontController.php

```diff
@@ -138,108 +137,28 @@ public function airportIndex() {
         return view('site.airports.index')->with('airports', $airports);
     }
 
-    public function searchAirport(Request $request) {
```

Comment:
No issue removing this. I guess when I fixed the search function in the dashboard I forgot to fix it here. You're right - no one has ever brought this up, so safe to say the feature isn't being used.